spring-integration-debezium/src/test/resources/docker/mysql/inventory.sql

```diff
@@ -0,0 +1,124 @@
+INSERT INTO products_on_hand VALUES (102,8);
+INSERT INTO products_on_hand VALUES (103,18);
+INSERT INTO products_on_hand VALUES (104,4);
+INSERT INTO products_on_hand VALUES (105,5);
+INSERT INTO products_on_hand VALUES (106,0);
+INSERT INTO products_on_hand VALUES (107,44);
+INSERT INTO products_on_hand VALUES (108,2);
+INSERT INTO products_on_hand VALUES (109,5);
+
+# Create some customers ...
+CREATE TABLE customers (
+  id INTEGER NOT NULL AUTO_INCREMENT PRIMARY KEY,
+  first_name VARCHAR(255) NOT NULL,
+  last_name VARCHAR(255) NOT NULL,
+  email VARCHAR(255) NOT NULL UNIQUE KEY
+) AUTO_INCREMENT=1001;
+
+
+INSERT INTO customers
+VALUES (default,"Sally","Thomas","sally.thomas@acme.com"),
+       (default,"George","Bailey","gbailey@foobar.com"),
+       (default,"Edward","Walker","ed@walker.com"),
+       (default,"Anne","Kretchmar","annek@noanswer.org");
+
+# Create some fake addresses
+CREATE TABLE addresses (
+  id INTEGER NOT NULL AUTO_INCREMENT PRIMARY KEY,
+  customer_id INTEGER NOT NULL,
+  street VARCHAR(255) NOT NULL,
+  city VARCHAR(255) NOT NULL,
+  state VARCHAR(255) NOT NULL,
+  zip VARCHAR(255) NOT NULL,
+  type enum('SHIPPING','BILLING','LIVING') NOT NULL,
+  FOREIGN KEY address_customer (customer_id) REFERENCES customers(id)
+) AUTO_INCREMENT = 10;
+
+INSERT INTO addresses
+VALUES (default,1001,'3183 Moore Avenue','Euless','Texas','76036','SHIPPING'),
+       (default,1001,'2389 Hidden Valley Road','Harrisburg','Pennsylvania','17116','BILLING'),
+       (default,1002,'281 Riverside Drive','Augusta','Georgia','30901','BILLING'),
+       (default,1003,'3787 Brownton Road','Columbus','Mississippi','39701','SHIPPING'),
+       (default,1003,'2458 Lost Creek Road','Bethlehem','Pennsylvania','18018','SHIPPING'),
+       (default,1003,'4800 Simpson Square','Hillsdale','Oklahoma','73743','BILLING'),
+       (default,1004,'1289 University Hill Road','Canehill','Arkansas','72717','LIVING');
+
+# Create some very simple orders
+CREATE TABLE orders (
+  order_number INTEGER NOT NULL AUTO_INCREMENT PRIMARY KEY,
+  order_date DATE NOT NULL,
+  purchaser INTEGER NOT NULL,
+  quantity INTEGER NOT NULL,
+  product_id INTEGER NOT NULL,
+  FOREIGN KEY order_customer (purchaser) REFERENCES customers(id),
+  FOREIGN KEY ordered_product (product_id) REFERENCES products(id)
+) AUTO_INCREMENT = 10001;
+
+INSERT INTO orders
+VALUES (default, '2016-01-16', 1001, 1, 102),
+       (default, '2016-01-17', 1002, 2, 105),
+       (default, '2016-02-19', 1002, 2, 106),
+       (default, '2016-02-21', 1003, 1, 107);
+
+# Create table with Spatial/Geometry type
+CREATE TABLE geom (
+	id INTEGER NOT NULL AUTO_INCREMENT PRIMARY KEY,
+	g GEOMETRY NOT NULL,
+	h GEOMETRY);
+
+INSERT INTO geom
+VALUES(default, ST_GeomFromText('POINT(1 1)'), NULL),
+      (default, ST_GeomFromText('LINESTRING(2 1, 6 6)'), NULL),
+      (default, ST_GeomFromText('POLYGON((0 5, 2 5, 2 7, 0 7, 0 5))'), NULL);
+
+# Create a signalling table
+CREATE TABLE dbz_signal (
```

Comment:
Looks like we just add this table into DB.
Everything else is exactly the same what they have so far with their standard `debezium/example-mysql` image.
Why cannot we just call such a DDL manually after starting Testcontainer or even propagate there some extra script like they do in their Docker file: https://github.com/debezium/container-images/blob/main/examples/mysql/2.3/Dockerfile ?

Reply:
- Already tried to use MySQLContainer + debezium/example-mysql, but it complain that that the example-mysql is not a mysql image. 
- I don't like messing with DDLs inside the test code, given that we have applly customesd docker image.

Reply:
> but it complain that that the example-mysql is not a mysql image.

So, you didn't pay attention that I had that `.asCompatibleSubstituteFor("mysql")` in my change.

> I  don't like messing with DDLs inside the test code

And I don't like having files in the project which are really not related to the project.
That is going to be just a short script for extra table and same official image.

And from here I might prefer to not have that MSSQL integration test since there is no Docker image for it from Debezium repository.
Same snapshoting functionality is covered with other two databases.

The burden of future support of this staff may neglect the effort we spend on it giving the fact that there is no any tool in the project for end-users (yet).

I just want to be reasonable in what we are doing and why.